app/models/interactive_page.rb

```diff
@@ -167,18 +169,41 @@ def reportable_items
     visible_embeddables.select { |item| item.reportable? }
   end
 
-  # 2021-08-05 NP: Lets add embeddables this way still,
-  # Just look for a section index value
-  def add_embeddable(embeddable, position = nil, section_id = nil)
-    if(self.sections.empty?)
-      self.sections.create(Section::DEFAULT_PARAMS)
+  # 2021-08-05 NP: Lets continue to add embeddables this way and
+  ## look for a section identifier value
+  def add_embeddable(embeddable, position = nil, section_identifier = nil)
+
+    section_identifier ||= Section::DEFAULT_SECTION_TITLE
+
+    # Local function to test whether section_identifier is a numeric value
+    def numeric (x)
+      Float(x) != nil rescue false
     end
-    section = self.sections.find { |s| s.id == section_id } || self.sections.first
+
+    # look for a section specified by title or ID
+    if !numeric(section_identifier)
```

Comment:
Probably best in the future to flip the `!` case since there is an else for this conditional.